impl/src/main/java/org/jboss/weld/bootstrap/events/AbstractBeanDiscoveryEvent.java

```diff
@@ -75,6 +75,7 @@ protected TypeStore getTypeStore() {
     }
 
     protected BeanDeployment getOrCreateBeanDeployment(Class<?> clazz) {
-        return DeploymentStructures.getOrCreateBeanDeployment(deployment, getBeanManager(), bdaMapping, contexts, clazz);
+        return DeploymentStructures.getOrCreateBeanDeployment(deployment, getBeanManager(), bdaMapping, contexts,
+                clazz);
```

Comment:
Why this formatting change?

Reply:
Didn't do that intentionally, I suppose it's something to do with formatter plugin.
I'll revert it.